lib/tasks/migrate_user_organisation_to_user_permission.rake

```diff
@@ -0,0 +1,19 @@
+namespace :organisation_user_migrate do
+  task migrate_organisation_user_to_user_permission: :environment do
+    total_bm = Benchmark.measure do
+      UserPermission.insert_all(
```

Comment:
Worth adding `unique_by: %i[ user_id provider_id ]` to this?

Reply:
Added a unique by on the unique index 👍  Yes, this is probably going to be a one time script

Reply:
this looks like it has ben resolved via 
`unique_by: :index_user_permission_on_user_id_and_provider_id`